src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +63,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
```

Comment:
Remember to follow the repo's coding conventions and style guides! You can change the indentation rules for switch-cases in your IDE settings.